spec/factories.rb

```diff
@@ -1,9 +1,18 @@
-
 FactoryBot.define do
   factory(:user) do
     email { 'abc@example.com' }
     password { 'password' }
     password_confirmation { 'password' }
     username { 'john_doe' } 
   end
-end
\ No newline at end of file
+  
+  factory(:plant) do
+    description { Faker::Lorem.sentence }
+    name { Faker::Lorem.word }
+    climate { "Sunnytime" }
+    latin { "Aeschynanthus lobianus" }
+    category { "Hanging" }
+    ideallight { "Bright light" }
+    watering { "Keep moist between watering. Can be a bit dry between watering" }
+  end
+end
```

Comment:
@kokopuffz I have gone ahead to create the factory for the plant model, what is left is to create the plant_spec.rb and and the required test suites.